fabtests/common/shared.c

```diff
@@ -2979,21 +2979,6 @@ void ft_parseinfo(int op, char *optarg, struct fi_info *hints,
 		if (!strncasecmp("mr_local", optarg, 8))
 			opts->mr_mode &= ~FI_MR_LOCAL;
 		break;
-	case 'D':
-		if (!strncasecmp("ze", optarg, 2))
-			opts->iface = FI_HMEM_ZE;
-		else if (!strncasecmp("cuda", optarg, 4))
-			opts->iface = FI_HMEM_CUDA;
-		else
-			printf("Unsupported interface\n");
-		opts->options |= FT_OPT_ENABLE_HMEM | FT_OPT_USE_DEVICE;
-		break;
-	case 'i':
-		opts->device = atoi(optarg);
-		break;
-	case 'H':
-		opts->options |= FT_OPT_ENABLE_HMEM;
-		break;
```

Comment:
We should add ft_parse_hmem_opts in here so that anything that uses INFO_OPTS (which includes HMEM_OPTS) will also parse the hmem opts

Reply:
agree, I missed this one.

Reply:
Thanks for the review, Alexia. This has been addressed in latest changes.